File: js/sumPairs.js
function sumPairs(arr, num) {

    let pairs = [];

    for (let i = 0; i < arr.length; i++) {
        for (let j = i+1; j < (arr.length); j++) {
            if (arr[i] + arr[j] == num) {
                pairs.push([arr[i], arr[j]]);
            }; 
        };
    };

    if (pairs.length == 0) {
        return "unable to find pairs";
    } else {
        return pairs;
    }; 
};

console.log(sumPairs([1, 2, 3, 4, 5], 7))
module.exports = sumPairs;

File: python/sum_pairs.py
def sum_pairs(arr, num):
    
    pairs = []
    
    for i in range(len(arr)):
        for j in range(i + 1, len(arr)):
            if arr[i] + arr[j] == num:
                pairs.append([arr[i], arr[j]])
    
    if pairs == []:
        return "unable to find pairs"
    
    else:
        return pairs